app/views/publish/courses/fees_and_financial_support/_guidance.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= t("publish.providers.fees_and_financial_support.edit.guidance_html") %>
+<details class="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      <%= t("publish.providers.fees_and_financial_support.edit.summary_link") %>
+    </span>
+  </summary>
+  <div class="govuk-!-margin-top-4" />
+  <%= render Find::Utility::AdviceComponent::View.new(title: "Financial support from the government") do %>
```

Comment:
This has the namespace `Find` but we're using it in Publish views. Can we find a way to make this work?

Maybe remove it from the Find namespace into a shared one or something?

Reply:
Same for all the components and views below